app/src/main/java/org/p2p/wallet/settings/ui/mail/SettingsEmailConfirmPresenter.kt

```diff
@@ -56,18 +57,22 @@ class SettingsEmailConfirmPresenter(
     }
 
     private suspend fun updateDeviceShare() {
-        // TODO update with real logic
-        updateMetadata()
-        delay(5000)
-        view?.showSuccessDeviceChange()
-    }
-
-    private suspend fun updateMetadata() {
         val currentMetadata = metadataInteractor.currentMetadata ?: return
         val newMetadata = currentMetadata.copy(
             deviceShareDeviceName = DeviceInfoHelper.getCurrentDeviceName(),
             deviceNameTimestampSec = DateTimeUtils.getCurrentTimestampInSeconds()
         )
+        // refreshDeviceShare(newMetadata)
         metadataInteractor.updateMetadata(newMetadata)
+        delay(5000)
```

Comment:
you'll remove this after the method is ready?

Reply:
of course
146%
it is only for fake progress before we did not connect real torus method